packages/kit/src/runtime/server/endpoint.js

```diff
@@ -1,6 +1,6 @@
 import { to_headers } from '../../utils/http.js';
 import { hash } from '../hash.js';
-import { is_pojo } from './utils.js';
+import { is_pojo, normalized_request_method } from './utils.js';
```

Comment:
Most of our function names are verby — they describe what they do rather than what they return. I think this should be the same

```suggestion
import { is_pojo, normalize_request_method } from './utils.js';
```